content/en/docs/components/pipelines/v2/installation/quickstart.md

```diff
@@ -19,40 +18,41 @@ This tutorial helps you get started with a KFP deployment and a pipeline created
 
-    **Note:** This command installs KFP v2, which is in pre-release stage and is not yet stable. The v2 documentation is being developed continually and some of the links to the v2 documentation might be unavailable.
+* **The [kubectl](https://kubernetes.io/docs/tasks/tools/) command-line tool**: Install and configure your [kubectl context](https://kubernetes.io/docs/tasks/access-application-cluster/configure-access-multiple-clusters/) to connect with your cluster.
+
+* Run the following script to install the KFP SDK:
+  ```shell
+  pip install kfp --pre
+  ```
+
+  **Note:** This command installs KFP v2, which is in pre-release stage and is not yet stable. The v2 documentation is being developed continually and some of the links to the v2 documentation might be unavailable.
 
 After you complete the prerequisites, click each step to view the instructions:
 
 <details>
-  <summary><a name="kfp_qs_step1"></a><h2 style="display:inline;">Step 1: Deploy a KFP standalone instance into your cluster</h2></summary>
-  <hr/>
-  This step shows how to deploy a KFP standalone instance into an existing Kubernetes cluster.
+<summary><a name="kfp_qs_step1"></a><h2 style="display:inline;">Step 1: Deploy a KFP standalone instance into your cluster</h2></summary>
 
-  Run the following script after replacing `PIPELINE_VERSION` with the desired version of KFP:
+This step demonstrates how to deploy a KFP standalone instance into an existing Kubernetes cluster.
 
-  ```shell
-  export PIPELINE_VERSION="2.0.0-alpha.4"
+Run the following script after replacing `PIPELINE_VERSION` with the desired version of KFP (release are listed [here][pypi]):
 
-  kubectl apply -k "github.com/kubeflow/pipelines/manifests/kustomize/cluster-scoped-resources?ref=$PIPELINE_VERSION"
-  kubectl wait --for condition=established --timeout=60s crd/applications.app.k8s.io
-  kubectl apply -k "github.com/kubeflow/pipelines/manifests/kustomize/env/dev?ref=$PIPELINE_VERSION"
-  ```
+```shell
+export PIPELINE_VERSION="2.0.0-alpha.4"
 
-  After you deploy Kubernetes, obtain your KFP endpoint by following [these instructions][installation].
-  <!-- TODO: add more precise section link and descriptive link text (with more context) when available -->
+kubectl apply -k "github.com/kubeflow/pipelines/manifests/kustomize/cluster-scoped-resources?ref=$PIPELINE_VERSION"
+kubectl wait --for condition=established --timeout=60s crd/applications.app.k8s.io
+kubectl apply -k "github.com/kubeflow/pipelines/manifests/kustomize/env/dev?ref=$PIPELINE_VERSION"
+```
+
+After you deploy Kubernetes, obtain your KFP endpoint by following [these instructions][installation].
+<!-- TODO: add more precise section link and descriptive link text (with more context) when available -->
 </details>
+
 <details>
-  <summary><a name="kfp_qs_step2"></a><h2 style="display:inline;">Step 2: Create and run a simple pipeline using the KFP SDK</h2></summary>
-  <hr/>
```

Comment:
The `hr` tag broke paragraph formatting and wasn't putting line breaks into the subsequent text correctly. Hugo can be finicky like this sometimes.